.goreleaser.yml

```diff
@@ -47,7 +47,7 @@ builds:
     flags:
       - -trimpath
     main: ./cmd/step-cloudkms-init/main.go
-    binary: bin/step-cloudkms-init
+    binary: step-cloudkms-init
```

Comment:
These other binaries are now deprecated, we will not remove the code until the next release, but we can probably remove them from the packages.